src/SqlClient.DesignTime/DesignTimeConnectionString.fs

```diff
@@ -68,7 +68,7 @@ type internal DesignTimeConnectionString =
                     value
                 else
                     let section = ConfigurationManager.ConnectionStrings.[name]
-                    if section = null 
+                    if isNull section
```

Comment:
While we fix code here and there like this, you can consider doing minor reformatting edits; such as putting the `then` at same line as the `if` and similar adjustments to make the code, just per your taste, so long it is not invasive (not for whole file, just the location / block that is edited).

Reply:
isNull just came with a more recent FSharp.Core. I tried to avoid changing files too much, just so it's easy to focus on the main goals.

Reply:
Done.